addons/arsenal/XEH_preInit.sqf

```diff
@@ -8,6 +8,7 @@ PREP_RECOMPILE_START;
 PREP_RECOMPILE_END;
 
 // Arsenal
+GVAR(defaultLoadoutsList) = [];
```

Comment:
You're breaking default loadouts, this variable already exists whatever happens due to https://github.com/acemod/ACE3/blob/master/addons/arsenal/Cfg3DEN.hpp#L11